src/components/About/About.js

```diff
@@ -0,0 +1,23 @@
+import "./About.css";
+import Image from "../../images/profile.jpg";
```

Comment:
`import Image from "../../images/profile.jpg";
`

* The project won't start. See the erros here http://joxi.ru/823Y6BpSzY3JYr 


there is no `profile.jpg` in `images`